### ISSUE TITLE
Markdown visual bugs (bold text on 3rd level in lists)

### ISSUE THREAD

**Issue by nikitasius** [a85/POSTMan-Chrome-Extension#4967]
**Describe the bug**
Incorrect markdown interpretation in visual editor

**To Reproduce**
Write a list with following text (in a request description):
```md
* 1st level **visual is fine**
  * 2nd level **visual is fine**
    * 3rd level **visual isn't feelint itself good** blablabla
```

**Expected behavior**
Correct markdown

**Screenshots**
Here is in editor:
![image](https://user-images.githubusercontent.com/3670331/43679523-af491300-9826-11e8-9339-4366ba11c461.png)
Here is saved (fine):
![image](https://user-images.githubusercontent.com/3670331/43679527-b9a58c84-9826-11e8-9e96-cbc2ed8f8ca1.png)

**App information (please complete the following information):**
 - Native App
 - Postman v6.2.3
 - OS: 4.9.0-0.bpo.5-amd64 #1 SMP Debian 4.9.65-3+deb9u2~bpo8+1 (2017-01-05) x86_64 GNU/Linux

**Comment by prashantagarwal** [comment]
@nikitasius Seems to be issue with tab space vs normal space. Try indenting using tabs and see if it works for you.

**Comment by nikitasius** [comment]
Yep:
![image](https://user-images.githubusercontent.com/3670331/43715422-c66565d8-9980-11e8-8a1c-3fc7768d3c21.png)
___
![image](https://user-images.githubusercontent.com/3670331/43715440-cf145b58-9980-11e8-87e5-1637ba25a08f.png)
___
With tabs it display correct, but in original markdown you just need 2 spaces for new level. Spaces are more handy and visually pretty (due smaller size).

**Comment by prashantagarwal** [comment]
@nikitasius I am closing this for now since the workaround seems to be working. We are also looking into improving core text editor and I have linked this issue with the same.

**Comment by kamalaknn** [comment]
I'm reopening this issue since it is easy to track the fixes for individual issues. We will keep this thread up to date with the status of the fix.

**Comment by pankajk07** [comment]
Fixed as part of v7.24
